packages/components/src/components/icon-icofont/component.tsx

```diff
@@ -13,7 +13,7 @@ type RequiredProps = AriaLabel & {
 type OptionalProps = {
 	part: string;
 };
-export type Props = Generic.Element.Members<RequiredProps, OptionalProps>;
+// type Props = Generic.Element.Members<RequiredProps, OptionalProps>;
```

Comment:
Kann das nicht gleich richtig weg oder wird es demnächst benötigt?

Reply:
Ja, auskommentierter Code - man übersieht es halt leicht, wenn man es für die Tests braucht.

In den Tests braucht man Props und States immer.

Bei icon-icofont ist das natürlich sinnlos, da die Komponenten verschwindet.